css/properties/content.json

```diff
@@ -110,6 +110,80 @@
             }
           }
         },
+        "gradient": {
+          "__compat": {
+            "description": "<code>&lt;gradient&gt;</code>",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/gradient",
+            "spec_url": "https://drafts.csswg.org/css-images-4/#gradients",
+            "support": {
+              "chrome": {
+                "version_added": "26"
```

Comment:
If this is a new feature, then it seems unrealistic that Chrome would support it as early as version 26. Do you have a source for this info?

Reply:
Yes it is just new Firefox. 
This [codepen](https://cdpn.io/pen/debug/MWPPKEj) is what I used for the testing and went back to Chrome 61 (as far as it would go).
Prior to me looking into this there was no BCD for `gradient` or `image-set`

Reply:
Okay, let's go with 26 then. I couldn't find any other information, so maybe you are right, it was always in Chrome.